tests/unit/specification/grammar_test.py

```diff
@@ -62,7 +63,7 @@ def parse_session(string: str) -> model.Session:
     if diagnostics_to_error(unit.diagnostics, error, STDIN):
         error.propagate()
     assert isinstance(unit.root, lang.SessionDecl)
-    return create_session(unit.root, ID("Package"), Path("<stdin>"))
+    return create_session(unit.root, ID("Package"), Path("<stdin>"), error)
```

Comment:
`error.propagate()`  should be called after `create_*`, otherwise any error in `create_*` will be just ignored.

Reply:
Fixed